### PR TITLE
Changed the qty and value fields type to allow for decimal values

### PR DESCRIPTION
**Pull request by jonathancod**
Changed the qty type from Number to Any to allow for decimal values on the hour count.

### CODE REVIEW

**Comment by jonathancod** [comment]
@rkmax  También @vctr90 hizo el cambio para permitir decimales en el Valor de la hora.

File: model.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ const modelHeader = tcomb.struct(
 const modelItem = tcomb.struct(
     {
         description: tcomb.String,
-        qty: tcomb.Integer,
-        value: tcomb.Number
+        qty: tcomb.Any,
```
Comment (rkmax) [inline model.js:21]: What do you think instrad Any use Number on both?, number support decimal

Comment (jonathancod) [reply model.js:21]: It has been almost a year, and I can't remember why I didn't use number instead of Any, but I'm sure there was a reason. xD

**Comment by rkmax** [comment]
I'll close this one, to avoid Any and allow decimals

**Pull request closed without merging**